app/utils/generic.py

```diff
@@ -216,7 +216,7 @@ def platform_specific_open(path: str | Path) -> None:
     path = str(path)
     if sys.platform == "darwin":
         logger.info(f"Opening {path} with subprocess open on MacOS")
-        if p.is_file() or (p.is_dir() and p.suffix == ".app"):
```

Comment:
I think this is likely better behavior, to actually open files instead of where they are located in finder. However, I just want to confirm that this doesn't break behavior in previous uses.

Also, the docstring needs to be updated to reflect this change. Granted, I don't think the doc string seems to be super accurate even without this change.